resources/patches/debian/system/gtk2.patch

```diff
@@ -1,13 +0,0 @@
-description: add missing include needed to build with gtk2
```

Comment:
Was the GTK2 patch removed in Debian?

Reply:
Yeah. It seems they fixed the gtk2 build error upstream